raven-android/src/main/java/com/getsentry/raven/android/event/helper/AndroidEventBuilderHelper.java

```diff
@@ -119,10 +123,36 @@ public void helpBuildingEvent(EventBuilder eventBuilder) {
         osMap.put("kernel_version", KERNEL_VERSION);
         osMap.put("rooted",         isRooted());
 
+        // App
+        PackageInfo packageInfo = getPackageInfo(ctx);
+        if (packageInfo != null) {
+            appMap.put("app_version", packageInfo.versionName);
+            appMap.put("app_build", packageInfo.versionCode);
+            appMap.put("app_identifier", packageInfo.packageName);
+        }
+
+        appMap.put("app_name", getApplicationName(ctx));
```

Comment:
👍 Cool, just wanted to confirm that was on purpose. (I don't have a strong feeling one way or another.)